eclipse-collections-api/src/main/java/org/eclipse/collections/api/map/MutableMapIterable.java

```diff
@@ -66,6 +67,19 @@ default V add(Pair<? extends K, ? extends V> keyValuePair)
      */
     V removeKey(K key);
 
+    /**
+     * Remove entries from the map at the specified {@code keys}.
+     *
+     * @return @return <tt>true</tt> if this map changed as a result of the call
```

Comment:
Double javadoc `@return @return`

Reply:
I see I had that mistake in my example, sorry

Reply:
fixed